packages/hash/api/src/model/knowledge/linkEntity.model.ts

```diff
@@ -0,0 +1,187 @@
+import {
+  GraphApi,
+  Entity,
+  LinkEntityMetadata,
```

Comment:
This is a big ask (sorry 🙃) but can we look across this PR and update all the relevant imports to pull from the subgraph lib.

Reply:
Thanks Alfie, have done so in https://github.com/hashintel/hash/pull/1435/commits/7dd96c542c7645cbd835a535564da9033d838024. Whilst using the `@hashintel/hash-subgraph` I encountered two issues:

1. The entity metadata types are currently not exported from the package, so I've created [a task](https://app.asana.com/0/1202805690238892/1203409252899196/f) to address this
2. The utility methods aren't exposed in the root, so I have to import them using `@hashintel/hash-subgraph/src/stdlib/element/entity` - wasn't sure if you were intending on addressing this, should I create a task for this as well?

Reply:
> wasn't sure if you were intending on addressing this, should I create a task for this as well?

I'll [create a relevant task](https://app.asana.com/0/1202805690238892/1203410068351216/f), thanks for highlighting. It's not clear if we do want to completely change this, I think Alex has some opinions on how we should be designing public API of typescript packages, it would seem the ecosystem is starting to support this a lot better (the `exports` keyword in `package.json` now)

We can revisit this when we're ready to (or choose to) actually publish the package.

Reply:
https://github.com/hashintel/hash/pull/1451 For the first point